docs/source/bag-overview.rst

```diff
@@ -0,0 +1,31 @@
+Overview
+========
+
+*Bag* is an abstract collection, like *list* or *set*.  It is a friendly
+synonym to multiset_. A bag or a multiset is a generalization of the concept
+of a set that, unlike a set, allows multiple instances of the multiset's
+elements.
+
+* ``list``: *ordered* collection *with repeats*, ``[1, 2, 3, 2]``
+* ``set``: *unordered* collection *without repeats*,  ``{1, 2, 3}``
+* ``bag``: *unordered* collection *with repeats*, ``{1, 2, 2, 3}``
+
+So a bag is like a list, but it doesn't guarantee an ordering among elements.
+There can be repeated elements but you can't ask for a particular element.
+
+.. _multiset: http://en.wikipedia.org/wiki/Bag_(mathematics)
+
```

Comment:
I moved the bag execution section back into the overview page. So now the sections are: Overview, Examples, Create Dask Bags, API. We can continue to iterate here along with the other collection sections.